src/domain/page/pageHeader/PageHeaderContainer.tsx

```diff
@@ -11,6 +11,8 @@ import List from '../../../common/list/List';
 import ListItem from '../../../common/list/ListItem';
 import { useCurrentUser } from '../../auth/hooks';
 import authService from '../../auth/authService';
+import Text from '../../../common/text/Text';
+import styles from './pageHeader.module.scss';
```

Comment:
I don't recall why this was called a container in the first place or the implementation has been changed somewhere down the line, but now especially that we are applying styles it looks more of a presentational component. So do you think it would be better to rename it? 
`PageHeaderContainer.tsx` => `PageHeader.tsx`

Reply:
Makes sense, let's do it